src/Microsoft.ML.Data/Transforms/ConversionsExtensionsCatalog.cs

```diff
@@ -126,4 +128,29 @@ public static KeyToValueMappingEstimator MapKeyToValue(this TransformsCatalog.Co
             IComponentFactory<IMultiStreamSource, IDataLoader> loaderFactory = null)
             => new ValueToKeyMappingEstimator(CatalogUtils.GetEnvironment(catalog), columns, file, termsColumn, loaderFactory);
     }
+
+    /// <summary>
+    /// Extensions for the ValueMapping Estimator
+    /// </summary>
+
+    public static class ToMappedValueCatalog
+    {
```

Comment:
no need for them to be on a separate class.

Reply:
I moved it up

---
In reply to: [243493007](https://github.com/dotnet/machinelearning/pull/1710#discussion_r243493007) [](ancestors = 243493007)